ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -329,7 +383,8 @@ object BigDiffy extends Command {
       s"""BigDiffy - pair-wise field-level statistical diff
         |Usage: ratatool $command [dataflow_options] [options]
         |
-        |  --mode=[avro|bigquery]
```

Comment:
do we want to support `mode` as well for backwards compatibility

Reply:
I think it's fine to break it, will add a more helpful log error if `mode` is passed in